src/Symfony/Component/Validator/Constraints/NullValidator.php

```diff
@@ -18,6 +18,8 @@
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
  * @api
+ *
+ * @deprecated
```

Comment:
Per @stof "The NullValidator needs to be deprecated too"

@stof are we deprecating this or not?